docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -40,26 +24,24 @@ networkers=(
 )
 ----
 +
-** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the `Networker` node and its IP address.
+** Replace `["standalone.localdomain"]="192.168.122.100"` with the name of the Networker node and its IP address.
```

Comment:
** Replace `["standalone.localdomain"]="192.168.122.100"` with the name and IP address of the Networker node.